config/osti_release_input_schematron.xml

```diff
@@ -106,7 +106,7 @@
             <assert test="text()">
             If exist, authors field may not be empty.
             </assert>
-            <assert test="count(*) >= 1 and count(first_name) = 1 and count(last_name) = 1">The element author should have 'first_name' and 'last_name' tags.</assert>
+            <assert test="count(*) >= 1 and (count(first_name) = 1 and count(last_name) = 1) or (count(full_name) = 1)">The element author should have 'first_name' and 'last_name' or 'full_name' tags.</assert>
```

Comment:
This was our first schematron.   Ron had created a new one.  We will swap this new one with the modification to remove the binding version since Python does not work with version 2.

Reply:
Ok, that sounds good.